README.md

```diff
@@ -90,10 +90,13 @@ The [Helm chart](https://artifacthub.io/packages/helm/metrics-server/metrics-ser
 
 ## Security context
 
-Metrics Server requires the `CAP_NET_BIND_SERVICE` capability in order to bind to a privileged ports as non-root.
-If you are running Metrics Server in an environment that uses PSPs or other mechanisms to restrict pod capabilities, ensure that Metrics Server is allowed
-to use this capability.
-This applies even if you use the `--secure-port` flag to change the port that Metrics Server binds to to a non-privileged port.
+Metrics Server may require the `CAP_NET_BIND_SERVICE` capability, in order to bind to a privileged port as non-root.
```

Comment:
It works to me, thanks for the update.  `CAP_NET_BIND_SERVICE` capability troubled us for a loog time.